### PR TITLE
Hound enabling

### PR DESCRIPTION
**Pull request by xavijam**
Enabling [hound](https://houndci.com/) for SCSS files. Also take into account that enabling scss-lint in your IDE is almost a must.
· [Linter properties documentation](https://github.com/brigade/scss-lint/blob/master/lib/scss_lint/linter/README.md)

@CartoDB/frontend

### CODE REVIEW

**Comment by matallo** [comment]
:+1:

File: .scss-lint.yml [hunk 1]
```diff
@@ -0,0 +1,168 @@
+scss_files: "**/*.css.scss"
```
Comment (viddo) [inline .scss-lint.yml]: I've already been runing scss-lint for some time, but I've got `scss_files: 'app/assets/stylesheets/**/*.css.scss'` here, is there any reason to include other dirs?

Comment (xavijam) [inline .scss-lint.yml]: Yeah, maybe with that path is enough

**Comment by juanignaciosl** [comment]
Retest this, please.

**Comment by xavijam** [comment]
@javierarce and I have been thinking about the problem we will have with linter in the old styles, like table, map etc, so we should exclude them once we have to edit them, do you agree?

**Comment by viddo** [comment]
If they are intended to be removed (or rather replaced by new design), I'm inclined to say yes. But in general and for style files that won't be removed in future design changes I believe [the FAQ](https://houndci.com/faq) is the right way to go:
> We believe the right approach is to convert a codebase over time, one pull request at a time, to conform to a style guide. Only focus on style in diffs and make smaller changes that ideally have tests associated with them.

**Comment by xavijam** [comment]
retest this please

**Comment by xavijam** [comment]
Yeah @viddo, we read that @javierarce and me yesterday. But the problem is, imagine, if you have to change the [table.css.scss](https://github.com/CartoDB/cartodb/blob/master/app/assets/stylesheets/table/table.css.scss) (just 730 lines) and our SCSS linter is enabled... it would be a nightmare!. We will change/make better that stylesheet, for example, so I think the best way is exclude it now or exclude it when it is gonna be touched.

**Pull request merged**